itl/command-plugins.conf

```diff
@@ -1780,6 +1785,96 @@ object CheckCommand "snmp" {
 			set_if = "$snmp_perf_oids$"
 			description = "Label performance data with OIDs instead of --label's"
 		}
+		// SNMP v2c options
+		"-C" = {
+			value = "$snmp_community$"
+			description = "Optional community string for SNMP communication (default is 'public')"
+			set_if = {{
+				if ((typeof(snmpversion) == Number && snmpversion <= 2) || (typeof(snmpversion) == String && (snmpversion == "1" || snmpversion == "2") || snmpversion == "2c")) {
+					return true
+				} else {
+					return false
+				}
```

Comment:
There is no need to make this whole block an `if` statement that just returns the if's condition.

Instead of
```
if (__FOO__) {
  return true
} else {
  return false
}
```
just write
```
return __FOO__
```

Same applies for the other similar cases below. Doing so would also shrink this PR's change size :)